tests/OSPSuite.R.Tests/Services/DataImporterSpecs.cs

```diff
@@ -51,19 +21,19 @@ public class When_importing_data_from_r : concern_for_DataImporter
       [Observation]
       public void should_import_simple_data()
       {
-         var _importerConfiguration = getConfiguration("importerConfiguration1.xml");
-         var dataRepositories = sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings, getFileFulName("sample1.xlsx"));
+         sut.ImportXslxFromConfiguration(getFileFulName("importerConfiguration1.xml"), getFileFulName("sample1.xlsx")).Count.ShouldBeEqualTo(1);
```

Comment:
yeah, that is finally what I expected from the very beginning :)
There are more use cases for sure, but this one (importing a data file based on existing configuration) is the most important for me.